lib/index.js

```diff
@@ -52,14 +53,60 @@ function compileBody(str, options){
   var debug_sources = {};
   debug_sources[options.filename] = str;
   var dependencies = [];
+  var plugins = options.plugins || [];
+  plugins = plugins.map(function (plugin) {
+    if (typeof plugin === 'string') {
+      var opts = {
+        basedir: path.dirname(module.parent.filename)
+      };
+      // Test if plugin name is local (i.e. `./my-plugin`)
+      // Regex from https://github.com/substack/node-resolve/blob/5cae82fb22cb64d5b72f703c787dc0fd418ed412/lib/sync.js#L21
+      if (/^(?:\.\.?(?:\/|$)|\/|([A-Za-z]:)?[\\\/])/.test(plugin)) {
+        return require(resolve(plugin, opts))();
```

Comment:
This seems a little bit brittle.  What if multiple modules call `require('pug')`?  I think it might be better to require absolute paths for local modules (or paths relative to current working directory).